neural_compressor/torch/quantization/load_entry.py

```diff
@@ -25,43 +25,85 @@
     RTNConfig,
     TEQConfig,
 )
+from neural_compressor.torch.utils import LoadFormat
 
 config_name_mapping = {
     FP8_QUANT: FP8Config,
 }
 
 
-def load(output_dir="./saved_results", model=None):
-    """The main entry of load for all algorithms.
+def load(model_name_or_path, original_model=None, format="default", device="cpu", **kwargs):
+    """Load quantized model.
 
-    Args:
-        output_dir (str, optional): path to quantized model folder. Defaults to "./saved_results".
-        model (torch.nn.Module, optional): original model, suggest to use empty tensor.
+    1. Load INC quantized model in local.
```

Comment:
nit: Please align the introduction and cases. For example:
1. INC 
      case 1: WOQ + local
      case 2: INT8/FP8 + local
      ...
2. HF
      ...

Reply:
update in https://github.com/intel/neural-compressor/commit/a33c766e8865f009ab02fa4ccd24cedd85574113